src/components/Col/Col.tsx

```diff
@@ -54,14 +54,14 @@ const sizeStyle = (props: Types.StyleProps & Types.ColProps) => {
 const offsetStyle = ({ theme, offset }: Types.StyleProps & Types.ColProps) =>
   offset &&
   typeof offset === 'object' &&
-  constants.BREAKPOINTS.map(
+  constants.BREAKPOINTS.filter((breakpoint) => offset[breakpoint]).map(
     (breakpoint) =>
       offset[breakpoint] &&
       css`
         ${media(breakpoint)} {
-          margin-left: ${offset[breakpoint] > 0
```

Comment:
Maybe we need to keep this verification only to avoid negative values, what do you think?

`offset[breakpoint] >= 0`

Reply:
Good point. I've added the verification.